mypy/test/data.py

```diff
@@ -191,6 +193,9 @@ def __init__(self,
         self.file = file
         self.writescache = writescache
         self.only_when = only_when
+        if ((platform == 'windows' and sys.platform != 'win32')
+                or (platform == 'posix' and sys.platform not in ('linux', 'darwin'))):
```

Comment:
But what about BSD? I think the only reasonable test for POSIX is "not Windows".